poofarmer/.mono/metadata/ide_messaging_meta.txt

```diff
@@ -0,0 +1,2 @@
+﻿51141
+D:/Godot_v3.5.1/Godot_v3.5.1-stable_mono_win64.exe
```

Comment:
Can you run `git rm --cached *.mono*`?

This file will be ignored in future commits, but we should probably not have it in here now because it is user specific